rali/rali_pybind/example/tf_detection_pipeline.py

```diff
@@ -79,10 +104,29 @@ def main():
 	pipe.build()
 	
 	imageIterator =  RALIIterator(pipe)
-	for i, (images_array, bb, labels_array) in enumerate(imageIterator, 0):
-		print("\nIMAGES ARRAY:\n",images_array)
-		print("\nBBOXS ARRAY:\n",bb)
-		print("\nLABELS ARRAY:\n",labels_array)
+	sum = 0
```

Comment:
Codacy found an issue: [Redefining built-in 'sum'](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6302037)